src/checkpoint/checkpoint_private.h

```diff
@@ -12,6 +12,23 @@
 #define WTI_CKPT_FOREACH_NAME_OR_ORDER(ckptbase, ckpt) \
     for ((ckpt) = (ckptbase); (ckpt)->name != NULL || (ckpt)->order != 0; ++(ckpt))
 
+/*
+ * WTI_CKPT_HANDLE_STATS --
+ *     Statistics related to handles.
+ */
+struct __wti_ckpt_handle_stats {
+    uint64_t apply;           /* handles applied */
+    uint64_t apply_time;      /* applied handles gather time */
+    uint64_t drop;            /* handles dropped */
```

Comment:
While we're here could you please improve this comment. I've been mislead by this implying dropping handles but it's actually dropping checkpoints for that handle. So I would replace it with:
`handle checkpoints dropped.

Reply:
Does it mean we need to update the other fields too?

Reply:
No, updating the comment here where we define it is okay. If you encounter it elsewhere I expect the comment where it's defined to be what clarifies it.